src/common/http_header.cc

```diff
@@ -340,15 +341,33 @@ Host::Host(const std::string& data)
 
 void
 Host::parse(const std::string& data) {
```

Comment:
Please add a set of unit tests (including sending in garbage data) for Host::parse(), or extend any existing unit test to cover IPV6 addresses and mal-formed IPV4 and IPV6 addresses.

Reply:
Will do